src/plugins/data/public/ui/query_editor/query_editor.tsx

```diff
@@ -85,9 +109,12 @@ export default class QueryEditorUI extends Component<Props, State> {
     index: null,
     suggestions: [],
     indexPatterns: [],
+    isCollapsed: true,
```

Comment:
should be false at first right?

Reply:
fixed the naming. The default mode is expanding